lib/internal/event_target.js

```diff
@@ -463,6 +475,22 @@ class NodeEventTarget extends EventTarget {
     this.addEventListener(type, listener, { [kIsNodeStyleListener]: true });
     return this;
   }
+  emit(type, arg) {
+    if (arguments.length > 2) {
```

Comment:
I am not sure this is the right error to throw and I am not sure throwing is the right behavior.

Other alternatives can be:
 - Wrap it in an array
 - Ignore additional arguments and emit the first one
 - Behave differently for emitter style and target style listeners.

Reply:
> Ignore additional arguments and emit the first one

I think I’d have a mild preference for this one, but really only a mild one.